src/Sulu/Component/Rest/FlattenExceptionNormalizer.php

```diff
@@ -36,6 +43,26 @@ public function normalize($exception, $format = null, array $context = [])
             $data['code'] = $exception->getCode();
         }
 
+        if (\in_array($this->environment, ['dev', 'test'])) {
+            $errors = '';
+
+            if ($exception instanceof FlattenException) {
+                $errors .= $exception->getClass();
+                $errors .= ': ';
+                $errors .= $exception->getMessage();
+                $errors .= ' in ';
+                $errors .= $exception->getFile();
+                $errors .= ':';
+                $errors .= $exception->getLine();
+                $errors .= \PHP_EOL . 'Stack trace:' . \PHP_EOL;
+                $errors .= $exception->getTraceAsString();
```

Comment:
Any reason not to use the [`getAsString` method](https://github.com/symfony/symfony/blob/5.1/src/Symfony/Component/ErrorHandler/Exception/FlattenException.php#L373)?

Reply:
Oh didn't saw that strange they didn't add a __toString function then. I used the `getAsString` method.